imessage-exporter/src/exporters/html.rs

```diff
@@ -181,14 +190,20 @@ impl<'a> Writer<'a> for HTML<'a> {
             );
 
             match message_part {
-                BubbleType::Text(text) => {
-                    self.add_line(
+                BubbleType::Text(text) => match text.starts_with(FITNESS_RECEIVER) {
```

Comment:
Any reason to use `match` over `if/else` here?

Reply:
I thought match was cleaner than having the lines with braces, but it does seem odd to match on `bool`

Reply:
Which style do you think is cleaner? 

```rust
match text.starts_with("https://music.apple") {
    true => Variant::App(CustomBalloon::Music),
    false => Variant::App(CustomBalloon::URL),
}
```

or 

```rust
if text.starts_with("https://music.apple") {
    Variant::App(CustomBalloon::Music)
} else {
    Variant::App(CustomBalloon::URL)
}
```

Reply:
This works:

```rust
text.starts_with("https://music.apple")
    .then_some(Variant::App(CustomBalloon::Music))
    .unwrap_or(Variant::App(CustomBalloon::URL))
```

But `clippy` doesn't like it:

```
warning: use of `.then_some(..).unwrap_or(..)` can be written more clearly with `if .. else ..`
   --> imessage-database/src/tables/messages.rs:570:29
    |
570 | / ...                   text.starts_with("https://music.apple")
571 | | ...                       .then_some(Variant::App(CustomBalloon::Music))
572 | | ...                       .unwrap_or(Variant::App(CustomBalloon::URL))
    | |______________________________________________________________________^ help: try: `if text.starts_with("https://music.apple") { Variant::App(CustomBalloon::Music) } else { Variant::App(CustomBalloon::URL) }`
    |
    = note: `#[warn(clippy::obfuscated_if_else)]` on by default
```